itou/templates/users/details.html

```diff
@@ -150,7 +150,39 @@ <h2>Données administratives</h2>
                                             <i class="text-disabled">Non renseigné</i>
                                         {% endif %}
                                     </li>
+                                    {% if profile.advisor_information %}
+                                        <div class="c-box c-box--structure c-info w-75" id="advisor-info-details-collapsable">
+                                            <div class="c-box--structure__summary" data-bs-toggle="collapse" data-bs-target="#advisorInfoDetails" aria-expanded="false" aria-controls="advisorInfoDetails">
```

Comment:
Faudrait-il émettre un évènement matomo pour pouvoir suivre les clics ?

Reply:
Oui ! Je l'avais noté dans le spec hier après l'ouverture de la PR 🙈

Reply:
Je ne suis pas arrivé voir les événements sur Matomo, mais j'ai ajouté sur le PR et les 3 URLs ont l'air bon ?

* `?e_c=gps&e_a=clic-coordonees-conseiller&e_n=30`
* `...e_n=57`
* `en_n=ailleurs`

Réponse 204

Reply:
Matomo est assez capricieux. Le plus facile est d’utiliser le journal de visites qui est alimenté en temps réel.

Je vois bien les éléments remonter :+1: 
![image](https://github.com/user-attachments/assets/149836ea-cf30-43ef-8a92-d28b7b52859d)